docling/pipeline/standard_pdf_pipeline.py

```diff
@@ -96,8 +96,15 @@ def __init__(self, pipeline_options: PdfPipelineOptions):
     def download_models_hf(
         local_dir: Optional[Path] = None, force: bool = False
     ) -> Path:
+        from functools import partialmethod
+
         from huggingface_hub import snapshot_download
 
+        # Disable tqdm prints used by HF
+        from tqdm import tqdm
+
+        tqdm.__init__ = partialmethod(tqdm.__init__, disable=True)
```

Comment:
we shouldn't monkey patch, the HF library has specific options for it already
https://huggingface.co/docs/huggingface_hub/v0.26.5/en/package_reference/file_download#huggingface_hub.snapshot_download.tqdm_class